### DISCUSSION TITLE
<title>Request takedown of an api key exposed in github

### DISCUSSION THREAD

**Discussion by anishgupadhya1997** [community/community#130828] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I need to request the takedown of an API key that is currently exposed on GitHub. How can I proceed with removing this sensitive information from the platform?

**Comment by Josekariz** [comment]
If I found an exposed API key in my repository, my immediate actions would be:

1. Remove the API Key: Edit the file to remove the exposed API key completely.
2. Update .gitignore: Add the file containing the API key to .gitignore to prevent it from being committed again.
3. Rotate the API Key: Generate a new API key and update my application's configurations to use the new key.
4. Review and Monitor: Check the repository's history to ensure the key is no longer accessible, and monitor for any potential misuse.
